packages/driver-adapter-utils/src/error.ts

```diff
@@ -0,0 +1,11 @@
+import { Error as ErrorObject } from './types'
+
+export class PrismaAdapterError extends Error {
+  name = 'PrismaAdapterError'
```

Comment:
They're called `Driver Adapters`, not `Prisma Adapters`. Let's use consistent naming conventions.

```suggestion
export class DriverAdapterError extends Error {
  name = 'DriverAdapterError'
```

Reply:
done